cpp/src/Ice/InputStream.cpp

```diff
@@ -13,12 +13,13 @@
 #include <Ice/UserExceptionFactory.h>
 #include <Ice/LocalException.h>
 #include <Ice/Protocol.h>
-#include <Ice/FactoryTableInit.h>
 #include <Ice/TraceUtil.h>
 #include <Ice/TraceLevels.h>
 #include <Ice/LoggerUtil.h>
 #include <Ice/SlicedData.h>
 #include <Ice/StringConverter.h>
+
+#include "Ice/FactoryTable.h"
```

Comment:
Since this is in `Ice` already, can we do:
```suggestion
#include "FactoryTable.h"
```

Reply:
It's in the public include directory, not next to this .cpp file.